docs/user/security/reporting.asciidoc

```diff
@@ -42,57 +55,66 @@ For more information, see {ref}/security-privileges.html[Security privileges].
 [role="screenshot"]
 image::user/security/images/reporting-privileges-example.png["Reporting privileges"]
 
-. Add space privileges.
+. Add space privileges for the {kib} applications that allow access to the reporting options.
++
+To allow users to create CSV reports in *Discover*, or PDF reports in *Canvas*,
+*Visualize Library*, and *Dashboard*, click  *Add Kibana privilege* for each application,
+then select the privileges to generate
+reports. For example, select *All* privileges for all features, or *Customize* to grant
+the privilege to generate reports for only specific applications.
++
+[role="screenshot"]
+image::user/security/images/reporting-custom-role.png["Reporting custom role"]
++
+[NOTE]
```

Comment:
I don't see this note in the text.

Reply:
That is strange. I see the note when I run the local docs server:
![image](https://user-images.githubusercontent.com/908371/115444306-3e42dd80-a1c9-11eb-8f90-5b7d0ca7c359.png)

Reply:
After merging, I checked that this note is indeed visible in the published docs: https://www.elastic.co/guide/en/kibana/master/secure-reporting.html